apps/trading/e2e/README.md

```diff
@@ -0,0 +1,118 @@
+# Trading End-To-End Tests
+
+This direcotry contains end-to-end tests for the trading application using vega-market-sim. This README will guide you through setting up your environment and running the tests.
+
+## Prerequisites
+
+- [Poetry](https://python-poetry.org/docs/#installing-with-the-official-installer)
+- [Docker](https://www.docker.com/)
+- [Python versions ">=3.9,<3.11"](https://www.python.org/)
+
+## Getting Started
+
+1. **Install Poetry**: Follow the instructions on the [official Poetry website](https://python-poetry.org/docs/#installing-with-the-official-installer).
+2. **Install Docker**: Follow the instructions on the [official Docker website](https://docs.docker.com/desktop/).
+3. **Install Python**: Follow the instructions on the [official Python website](https://www.python.org/)
+   **ensure you install a version between 3.9 and 3.11.**
+4. **Start up a Poetry environment**: Execute the commands below to configure the Poetry environment.
+
+```bash
+poetry shell
```

Comment:
hello, idiot here ;) I hear that you suggested it is obvious to be in the correct folder to run this command @mattrussell36 , however I just made this mistake. I think you lose nothing from being specific and pointing out you need to be in this folder. Definitely the kinda thing that made me say 'DUH' when I realised. But still